### ISSUE TITLE
Accessibility of Tables

### ISSUE THREAD

**Issue by ThePhisch** [rucio/webui#207]
The tables rows cannot be focused, hence they are only accessible via a mouse. This is a major breach of WCAG standards for keyboard accessibility, see [here](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=211#keyboard).

Proposed fix: in the `StreamedTable` generic table, find a solution to this. Allow users to tab/cycle through the table elements, improve accessibility.
